checks.d/directory.py

```diff
@@ -30,12 +30,12 @@ def check(self, instance):
 
         directory = instance["directory"]
         abs_directory = abspath(directory)
-        name = instance.get("name") or directory
-        pattern = instance.get("pattern") or "*"
-        recursive = instance.get("recursive") or False
-        dirtagname = instance.get("dirtagname") or "name"
-        filetagname = instance.get("filetagname") or "filename"
-        filegauges = instance.get("filegauges") or False
+        name = instance.get("name", directory)
+        pattern = instance.get("pattern", "*")
+        recursive = instance.get("recursive", False)
```

Comment:
This is a very common pitfall, can you fix this as well as `filegauges`
See https://github.com/DataDog/dd-agent/commit/0e862f5f7f16f5fc1466f4b8ebff1b4e44cac64e